main.cpp

```diff
@@ -160,75 +155,79 @@ struct Laundromat
         float waterUsed = 24.57f;
         float powUsed = 75.43f;
         int washes = 17;
-        void washClothes();
+        void wash();
         float countChang(int coins);
         float dwnTime(float timeOff);
     };
 
-    //1) wash clothes
     void washClothes();
-    //2) dry clothes
     void dryClothes();
-    //3) make change out of bills
     float makeChange(float price, float amPaid);
 };
-/*
-Thing 2) bank
-5 properties:
-    1) amount of money deposited into it(float)
-    2) amount of money withdrawn from it(float)
-    3) amount of interest paid(float)
-    4) amount of debt owed(float)
-    5) amount of principal paid(float)
-3 things it can do:
-    1) hold money
-    2) process transactions
-    3) produce a cash balance
- */
-struct Bank
+
+void Laundromat::washClothes()
+{
+
+}
+
+void Laundromat::dryClothes()
+{
+
+}
+
+float Laundromat::makeChange(float price, float amPaid)
+{
+    return amPaid - price;
+}
+
+void Laundromat::WashingMach::wash()
 {
-    //1) amount of money deposited into it(float)
 
+}
+
+float Laundromat::WashingMach::countChang(int coins)
+{
+    return coins;
+}
+
+float Laundromat::WashingMach::dwnTime(float timeOff)
+{
+    return timeOff;
+}
+
+struct Bank
+{
     float moneyDeposited = 200.25f;
-    //2) amount of money withdrawn from it(float)
     float moneyWithdrawn = 100.25f;
-    //3) amount of interest paid(float)
     float interestPaid = 03.03f;
-    //4) amount of debt owed(float)
     float debtOwed = 2000.05f;
-    //5) amount of principal paid(float)
     float Paid = 1500.25f;
-    //1) hold money
     float holdMoney(float amount);
-    //2) process transactions
     void transaction();
-    //3) produce a cash balance
     float cashBalance(float balance);
 };
```

Comment:
I didn't catch these improper function names in part 1. 
What you have written here does not match your plain-english actions.
You need to include the action words in your function names
```
void processTransactions();
float produceCashBalance(float balance);
```
- [ ] revise